app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
The URL formatting was carefully designed and I think we should stick to it
for API stability reasons; is it a problem to have a slash in a friendly_id
path?

On Jul 23, 2016 11:25 AM, "Ananya Maiti" <notifications@github.com> wrote:

> In app/controllers/map_controller.rb
> <https://github.com/publiclab/plots2/pull/600#discussion_r71973679>:
>
> > @@ -16,7 +16,10 @@ def index
> >    end
> >
> >    def show
> > -    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
> > +    @node = DrupalNode.find params[:name]
>
> @jywarren <https://github.com/jywarren> Same thing also holds for
> research notes. Should we use an url like /notes/user-07-23-2016-test-note
> for research notes?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/600/files/c968e35c5fa7b3232e5e5b2ca0bc9afa658586cd#r71973679>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ_J8_PhloWur8RKiFngTcZWecQyNks5qYjKGgaJpZM4JDL2B>
> .
>